src/main/java/danta/core/templating/AbstractTemplateContentModelImpl.java

```diff
@@ -0,0 +1,299 @@
+/**
+ * Danta AEM Bundle
```

Comment:
Hi @josecarlosgt, Could you fix the the bundle name, please?

Reply:
Done!